package.json

```diff
@@ -39,10 +39,13 @@
     "rollbar": "2.4.5",
     "swagger2": "1.0.1",
     "swagger2-koa": "1.0.2",
-    "winston": "3.1.0"
+    "winston": "3.1.0",
+    "@kubernetes/client-node": "0.5.3"
   },
   "devDependencies": {
-    "after-work.js": "4.3.2",
+    "@after-work.js/aw": "5.0.1",
```

Comment:
Is there a reason why we also bumped aw version?

Reply:
Yes, it included the functionality to mock whole files (so when you `require()` you get the mocked file) this was needed for our component tests to pass in a mocked version of the kubernetes lib

Reply:
Maybe we should scoop AW and only pull the parts that we need

Reply:
Do you know which parts that is?

Reply:
npm i -D @after-work.js/cli @after-work.js/node